src/frontend/ExceptionsDatePicker/ExceptionsDatePicker.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+import 'react-day-picker/lib/style.css';
+import { useField } from 'formik';
+import { SlotBookingRules } from '../../common/constants-common';
+import Modal from '../ui-components/Modal/Modal';
+import { format } from 'date-fns';
+
+interface ExceptionsDatePickerProps {
+  showDatePicker: boolean;
+  closeModal: (b: boolean) => void;
+}
+
+const ExceptionsDatePicker: React.FC<ExceptionsDatePickerProps> = ({
+  showDatePicker,
+  closeModal,
+}) => {
+  const [{ value: exceptions }, , { setValue }] = useField('exceptions');
```

Comment:
It's possible to use `useField<Array<string>>` here as well , so `exceptions` would be typed accordingly : )